python/lsst/ts/observatory/control/maintel/mtcs.py

```diff
@@ -841,9 +842,40 @@ async def close_m1_cover(self) -> None:
                 f"{MTMount.DeployableMotionState.DEPLOYED!r}"
             )
 
-    async def home_dome(self) -> None:
-        # TODO: Implement (DM-21336).
-        raise NotImplementedError("# TODO: Implement (DM-21336).")
+    async def home_dome(self, physical_az: float = 0.0) -> None:
+        self.log.info("Homing dome")
+        reported_az = await self.rem.mtdome.tel_azimuth.aget(timeout=self.fast_timeout)
+
+        offset = physical_az - reported_az.positionActual
+        self.log.debug(f"Dome azimuth offset: {offset} degrees")
+        target_az = Angle(self.home_dome_az - offset, unit=u.deg).deg
+        self.rem.mtdome.evt_azMotion.flush()
+        await self.rem.mtdome.cmd_moveAz.set_start(
+            position=target_az, velocity=0.0, timeout=self.park_dome_timeout
+        )
+
+        await self._handle_in_position(
+            self.rem.mtdome.evt_azMotion,
+            timeout=self.fast_timeout,
+            settle_time=self.tel_settle_time,
+            component_name="MTDome",
+        )
+
+        await self.rem.mtdome.cmd_stop.set_start(
+            engageBrakes=True, timeout=self.long_long_timeout
```

Comment:
you need to pass a `subSystemIds`. There's an enumeration for this you can use;

```
from lsst.ts.xml.enums.MTDome import SubSystemId
```

then;

```
      await self.rem.mtdome.cmd_stop.set_start(
            engageBrakes=True, subSystemIds =SubSystemId.AMCS, timeout=self.long_long_timeout
       )
```